tests/test_reader_binary.py

```diff
@@ -187,6 +185,11 @@ def _bad_params():
         b'\x69\x43\xA4\x0F\xE0\x82\x82\x87\x80\x9E',
         e_timestamp(_ts(2016, 2, 2, 0, 0, 30, off_hours=-7, precision=_PREC_SECOND))
     ),
+    (
+        b'\x69\xC0\x81\x81\x81\x80\x80\x80\xC7\x01',
+        e_timestamp(_ts(1, 1, 1, 0, 0, 0, None, precision=_PREC_SECOND, fractional_precision=7,
+                        fractional_seconds=Decimal('1e-7')))
```

Comment:
This is what I mean about `fractional_precision` and `fractional_seconds`, it seems you don't need both--as the decimal encodes the precision even for zero.  E.g.

```
>>> d = Decimal('1e-7')
>>> d.as_tuple().exponent
-7
```

Am I missing something?

Reply:
Changed. It was decided per the contract https://github.com/amzn/ion-python/pull/100#issuecomment-537212997 that `fractional_seconds` does not need `fractional_precision`.